dotnet-engine/Yggdrasil.Engine/FFI.cs

```diff
@@ -0,0 +1,152 @@
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace Yggdrasil;
+
+internal static class FFI
```

Comment:
I've moved this to a static class, there's no need to hold state for this, that's done by the engine pointer